Lib/json/__main__.py

```diff
@@ -0,0 +1,21 @@
+r"""Command-line tool to validate and pretty-print JSON
```

Comment:
I don't think this needs to be a raw string?
```suggestion
"""Command-line tool to validate and pretty-print JSON
```